packages/web/integrations/bridges/skip/queries.ts

```diff
@@ -0,0 +1,96 @@
+import {
+  Asset,
+  Chain,
+  MsgsRequest,
+  MsgsResponse,
+  RouteRequest,
+  RouteResponse,
+  TxStatusResponse,
+} from "./types";
+
+class SkipApiClient {
+  async assets({ chainID }: { chainID?: string } = {}) {
+    const urlParams = new URLSearchParams({
+      include_evm_assets: "true",
+    });
+
+    if (chainID) {
+      urlParams.set("chain_id", chainID);
+    }
+
+    const response = await fetch(
+      `https://api.skip.money/v1/fungible/assets?${urlParams.toString()}`
+    );
```

Comment:
I recommend using the [URL](https://developer.mozilla.org/en-US/docs/Web/API/URL) class to compose the URL since it includes the search params object internally and guarantees that the url is valid. 
```suggestion
    const url = new URL("/v1/fungible/assets", "https://api.skip.money")
    url.searchParams.append("include_evm_assets", "true")

    if (chainID) {
     url.searchParams.set("chain_id", chainID)
    }

    const response = await fetch(url.toString());
```